installer.sh

```diff
@@ -16,35 +16,29 @@
 
 export DEBIAN_FRONTEND=noninteractive
 
-#export PIPENV_VENV_IN_PROJECT="enabled"  #installs venv in same dir
-#export WORKON_HOME=/tmp
-export PIPENV_DOTENV_LOCATION=./.env
-export PIPENV_DONT_LOAD_ENV=0
-
 RED='\033[0;31m'
 GREEN='\033[0;32m'
 ORANGE='\033[0;33m'
+CYAN='\033[0;36m'
 BOLD='\033[1m'
 NC='\033[0m' # no color
 
 print_help() {
   echo -e "\\n""$CYAN""USAGE""$NC"
-  # echo -e "$CYAN-F$NC         Installation of EMBArk with all dependencies (typical initial installation)"
-  # echo -e "$CYAN-r$NC         Reinstallation of EMBArk with all dependencies (cleanup of docker environment first)"
-  # echo -e "$RED               ! This deletes all Docker-Images as well !""$NC"
-  echo -e "$CYAN-e$NC         Install EMBA only"
   echo -e "$CYAN-h$NC         Print this help message"
-  echo -e "$CYAN-d$NC         Build EMBArk"  # Webserver on host
-  echo
+  echo -e "$CYAN-d$NC         EMBArk default installation"
+  echo -e "$CYAN-F$NC         Installation of EMBArk for developers"
+  echo -e "$CYAN-e$NC         Install EMBA only"
+  echo -e "$CYAN-D$NC         Install for Docker deployment"
```

Comment:
Developer installation mode

Reply:
changed to `-F`, same as EMBA

Reply:
`-D` i planed for docker.
which isn't ready yet